### PR TITLE
feat: `DaskSelectorNamespace` 

### PR DESCRIPTION
**Pull request by FBruzzesi**
## What type of PR is this? (check all applicable)

- [ ] 💾 Refactor
- [x] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 🔧 Optimization
- [ ] 📝 Documentation
- [ ] ✅ Test
- [ ] 🐳 Other

## Related issues 

- Related issue #637 

## Checklist

- [x] Code follows style guide (ruff)
- [x] Tests added 
- [ ] Documented the changes

### CODE REVIEW

File: tests/test_selectors.py [hunk 1]
```diff
@@ -74,6 +66,16 @@ def test_categorical(request: Any, constructor: Any) -> None:
     compare_dicts(result, expected)
 
 
+def test_dask_categorical() -> None:
```
Comment (FBruzzesi) [inline tests/test_selectors.py:69]: Hack to test categorical selector until `cast` is not implemented

File: tests/test_selectors.py [hunk 2]
```diff
@@ -24,36 +24,28 @@
 }
 
 
-def test_selectors(constructor: Any, request: Any) -> None:
```
Comment (FBruzzesi) [inline tests/test_selectors.py:27]: This file diff gives me so much joy 😂

**Review by MarcoGorelli — APPROVED** [review]
yay thanks @FBruzzesi !

File: tests/test_selectors.py (hunk 1 above)
Comment (aidoskanapyanov) [reply tests/test_selectors.py:71]: No worries :) 2054694f09d70bbffc59a3390f4feaabb465f76b

**Pull request merged**